src/main/groovy/org/broadinstitute/orsp/consent/DataUseDTO.groovy

```diff
@@ -14,7 +14,7 @@ class DataUseDTO {
     String controlSetOption
     String gender
     Boolean pediatric
-    List<String> populationRestrictions
+    String populationRestrictions
```

Comment:
Is this really true? Have we restricted the autocomplete to be a single choice instead of multiple? It's hard to tell from the PR, but I don't see that specific change.

Reply:
It's true. This come from a change requested in the lucidchart. From a list to a free text. And then it was asked to get back to a list of ontologies.

Reply:
Then I'm confused ... if it is as you say
> And then it was asked to get back to a list of ontologies.
Then why is this changing from a list to a single string?

Reply:
It's a mistake that I made because I lost track of the back and forth changes. Should stay as a list.